src/components/Atoms/RichText/RichText.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+import DOMPurify from 'dompurify';
+
+export const RichTextInner = styled.div`
+  text-align: ${props => props.align};
+`;
+
+const RichText = ({ align, markup, ...rest }) => {
+  const sanitisedMarkup = DOMPurify.sanitize(markup);
+
+  return (
+    <RichTextInner
+      align={align}
+      dangerouslySetInnerHTML={{ __html: sanitisedMarkup }}
+      {...rest}
+    />
+  );
+};
+
+RichText.propTypes = {
+  align: PropTypes.string,
```

Comment:
here we could use `oneOf`
```
.oneOf([
    'left',
    'center',
    'right'
  ]),
```

Reply:
ahhh nice, I didn't know you could predefine props like that